lib/actuators/include/ContServo.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "Servo.h"
+
+class ContServo : public Servo {
+ public:
+  ContServo(PinName pin);
+  ContServo(PinName pin, float max_speed);
+  ContServo(PinName pin, float max_speed, std::chrono::duration<float> max_pulse,
+            std::chrono::duration<float> min_pulse);
+
+  /** Set the maximum speed of the servo
+   *
+   * @param max_speed The maximum speed of the servo
+   */
+  void setMaxSpeed(float max_speed);
+
+  /** Read the maximum speed of the servo
+   *
+   * @return The maximum speed of the servo
+   */
+  float getMaxSpeed(void) const;
+
+  /** Set the speed of the servo
+   *
+   * @param speed The speed of the servo (can be negative)
+   */
+  void setValue(float speed) override;
+  ContServo& operator=(float speed);
+
+  /** Read the current speed of the servo
+   *
+   * @return Current speed of servo
+   */
+  float getValue() const override;
+
+  // Override default period (ONLY USE FOR SPECIFIC FREQ REQUIREMENT)
+  void setPeriod(std::chrono::duration<float> period) {
+    m_pwm.period(period.count());
+  };
+
+ protected:
+  float m_max_speed,  // MAXIMUM ROTATION SPEED in ANGLES PER SECOND (from -max_speed to + max_speed),  only valid for
+                      // CONTINUOUS_SERVO types
+      m_speed;        // ROTATING SPEED in ANGLES PER SECOND
+  PinName m_pin;      // PIN the servo PWM signal is attached to
```

Comment:
So I think that the library code technically doesn't have to enforce units on values like speed and position, since they're just arbitrary values as far as the code is concerned. Should we leave it up to the user to decide what units they want to work in?

Reply:
it definetly should right. if i call setSpeed i need to know if i should give rad/sec or deg/sec

Reply:
originally id only read the first part of the comment so i was saying theres no need to put a comment saying max speed if ur var name is max speed. But, now, I see that the comment gives the units which is good. just do m_max_speed // deg/sec